src/actions/guides/frontend/rendering_html.cr

```diff
@@ -450,6 +450,25 @@ class Guides::Frontend::RenderingHtml < GuideAction
     # => true
     ```
 
+    ### Previous URL
+
+    Lucky also has a `previous_url` helper method you can use on your pages.
+
+    It returns the url of the page that issued the request (the referrer)
+    if possible, otherwise redirects to the provided default fallback
```

Comment:
```suggestion
    if possible, otherwise it uses the provided default fallback
```

Since I confused this with `redirect_back`, I figured it best to remove "redirect" language

Reply:
That makes sense. Thanks!